ios/Classes/PdftronFlutterPlugin.h

```diff
@@ -127,12 +149,20 @@ static NSString * const PTAnnotationFlagToggleNoViewKey = @"toggleNoView";
 
 + (PdftronFlutterPlugin *)registerWithFrame:(CGRect)frame viewIdentifier:(int64_t)viewId messenger:(NSObject<FlutterBinaryMessenger> *)messenger;
 
--(void)docVC:(PTDocumentViewController*)docVC annotationChange:(NSString*)xfdfCommand;
--(void)docVC:(PTDocumentViewController*)docVC bookmarkChange:(NSString*)bookmarkJson;
--(void)docVC:(PTDocumentViewController*)docVC documentLoaded:(NSString*)filePath;
+-(void)docVCBookmarkChange:(PTDocumentViewController*)docVC bookmarkJson:(NSString*)bookmarkJson;
+-(void)docVCExportAnnotationCommand:(PTDocumentViewController*)docVC xfdfCommand:(NSString*)xfdfCommand;
+-(void)docVCDocumentLoaded:(PTDocumentViewController*)docVC filePath:(NSString*)filePath;
+-(void)docVCDocumentError:(PTDocumentViewController*)docVC;
+-(void)docVCAnnotationChanged:(PTDocumentViewController*)docVC annotationsWithActionString:(NSString*)annotationsWithActionString;
+-(void)docVCAnnotationsSelected:(PTDocumentViewController*)docVC annotationsString:(NSString*)annotationsString;
+-(void)docVCFormFieldValueChanged:(PTDocumentViewController*)docVC fieldsString:(NSString*)fieldsString;
```

Comment:
Unsure on why the naming convention was changed here? A proposal:
```
documentController:(PTDocumentViewController*)docController bookmarksDidChange:(NSSgring*)bookmarkJson;
documentController:(PTDocumentViewController*)docController annotationsAsXFDFCommand:(NSString*)XFDFCommand;
documentController:(PTDocumentViewController*)docController documentLoadedFromFilePath:(NSString*)filePage;
documentController:(PTDocumentViewController*)docController documentError:(NSError*)error;
documentController:(PTDocumentViewController*)docController annotationsChangedWithActionString:(NSString*)actionString;
documentController:(PTDocumentViewController*)docController annotationsSelection:(NSString*)annotations;
documentController:(PTDocumentViewController*)docController formFieldValueChanged:(NSString*)fieldString;
```

Reply:
Was due to error not actually having an error parameter, which makes it hard to differentiate...